variables.tf

```diff
@@ -16,64 +16,72 @@
 
 variable "organization_id" {
   description = "The organization id for putting the policy"
-  default     = ""
+  type        = number
+  default     = null
 }
 
 variable "folder_id" {
   description = "The folder id for putting the policy"
-  default     = ""
+  type        = number
+  default     = null
 }
 
 variable "project_id" {
   description = "The project id for putting the policy"
-  default     = ""
+  type        = string
+  default     = null
 }
 
 variable "enforce" {
   description = "If boolean constraint, whether the policy is enforced at the root; if list constraint, whether to deny all (true) or allow all"
-  default     = ""
+  type        = bool
+  default     = null
 }
 
 variable "allow" {
   description = "(Only for list constraints) List of values which should be allowed"
-  type        = "list"
+  type        = list(string)
   default     = [""]
 }
 
 variable "deny" {
   description = "(Only for list constraints) List of values which should be denied"
-  type        = "list"
+  type        = list(string)
   default     = [""]
 }
 
 variable "exclude_folders" {
   description = "List of folders to exclude from the policy"
-  type        = "list"
+  type        = list(string)
   default     = [""]
```

Comment:
same

Reply:
Yes, if we want to stick to minimal changes
```
[11:27][user@host:~/workspace/google/terraform-google-org-policy/examples/list_org_exclude]$ terraform plan

Error: Missing required argument

  on main.tf line 28, in module "org-policy":
  28: module "org-policy" {

The argument "exclude_folders" is required, but no definition was found.


Error: Missing required argument

  on main.tf line 28, in module "org-policy":
  28: module "org-policy" {

The argument "deny" is required, but no definition was found.
```